### ISSUE TITLE
non .md files in folder causing issues

### ISSUE THREAD

**Issue by ajitbohra** [nikalsh/vuepress-auto-sidebar#2]
```
Error: ENOTDIR: not a directory, scandir 'Users/..../.gitignore'
```

**Comment by nikalsh** [comment]
Thanks for raising this issue, I will look into it. Can you show me what your docs folder structure looks like?

**Comment by nikalsh** [comment]
I just released a new version (1.1.0) which solves this. It now ignores all non-directories in the docs folder.
